src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -328,9 +291,46 @@ protected async Task UnregisterTimerAsync(IActorTimer timer)
         protected async Task UnregisterTimerAsync(string timerName)
         {
             await ActorRuntime.DaprInteractor.UnregisterTimerAsync(this.actorTypeName, this.Id.ToString(), timerName);
-            if (this.timers.ContainsKey(timerName))
+        }
+
+        internal MethodInfo GetMethodInfoUsingReflection(Type actorType, string callback)
+        {
+            return actorType.GetMethod(callback, BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.InvokeMethod);
```

Comment:
FYI - I think that `InvokeMethod` will be ignored here: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.bindingflags?view=net-5.0

Reply:
I guess so because we have restrictions on return type....so it anyways cannot be a constructor

Reply:
`GetMethod` only returns non-constructor methods regardless of that flag.